debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -48,7 +48,8 @@
     protected static final String TABLE_EXCLUDE_LIST_NAME = "table.exclude.list";
     protected static final String TABLE_WHITELIST_NAME = "table.whitelist";
     protected static final String TABLE_INCLUDE_LIST_NAME = "table.include.list";
-    private static final Pattern MSG_KEY_COLUMNS_PATTERN = Pattern.compile("^(([^:]+):([^:;\\s]+))+[^;]$");
+    private static final Pattern MSG_KEY_COLUMNS_PATTERN = Pattern.compile("^(([^:;]+):([^:;\\s]+))+$");
```

Comment:
Now that we already split up the expression at ";" before, this expression can be simplified a bit. In particular, the ";" can be removed from the negated classes.